contributors/devel/contributor-experience/contributor-github-cheat-sheet.en.md

```diff
@@ -0,0 +1,286 @@
+  - [Labels](#Labels)
+- [Working Locally](#Working-Locally)
+  - [Branch Strategy](#Branch-Strategy)
+    - [Adding Upstream](#Adding-Upstream)
+    - [Keeping Your Fork in Sync](#Keeping-Your-Fork-in-Sync)
+  - [Squashing Commits](#Squashing-Commits)
+
+
+## Communicating Effectively on GitHub
+
+
+### How to be Excellent to Each Other
+
+As a first step, familiarize yourself with the [Code of Conduct].
+
+
+#### Examples of Good/Bad Communication
+
+When raising an issue, or seeking assistance, please be polite with your request:
+
+  🙂 “X doesn’t compile when I do Y, do you have any suggestions?”
+
+  😞 “X doesn’t work! Please fix it!”
+
+When closing a PR, convey an explanatory and cordial message explaining
+why it does not meet the requirements to be merged.
+
+🙂 “I’m closing this PR because this feature can’t support the use case X. In
+   it's proposed form, it would be a better to be implemented with Y tool. Thank
+    you for working on this.”
+
+😞 “Why isn’t this following the API conventions? This should be done elsewhere!”
+
+
+## Submitting a Contribution
+
+### Opening and Responding to Issues
+
+GitHub Issues are the primary means of tracking things such as bug reports,
+enhancement requests, or reporting other issues such as failing tests. They are
+**not** intended for [user support requests]. For those, please check with the
+[troubleshooting guide], report the problem to [Stack Overflow] or follow up on
+the Kubernetes [User forum].
+
+**References:**
+- [Labels]
+- [Prow commands][commands]
+
+
+#### Creating an Issue
+
+- Use an issue template if one is available. Using the correct one will aid other
+  contributors in responding to your issue.
+  - Follow any directions described in the issue template itself.
+- Be descriptive with the issue you are raising.
+- Assign appropriate [labels]. If you are unsure, the [k8s-ci-robot][prow] bot
+  ([Kubernetes CI bot][prow]) will reply to your issue with the needed labels
+  for it to be effectively triaged.
+- Be selective when assigning issues using [`/assign @<username>`][assign] or
+  [`/cc @<username>`][cc]. Your issue will be triaged more effectively applying
+  correct labels over assigning more people to the issue.
+
+#### Responding to an Issue
+
+- When tackling an issue, comment on it letting others know you are working on
+  it to avoid duplicate work.
+- When you have resolved something on your own at any future time, comment on
+  the issue letting people know before closing it.
+- Include references to other PRs or issues (or any accessible materials),
+  example: _"ref: #1234"_. It is useful to identify that related work has been
+  addressed somewhere else.
+
+
+### Opening a Pull Request
+
+Pull requests (PR) are the main means of contributing code, documentation or
+other forms of work that would be stored within a git repository.
+
+**References:**
+- [Labels]
+- [Prow commands][commands]
+- [Pull request process]
+- [Github workflow]
+
```

Comment:
Should the list of References be repeated throughout the page?

Reply:
It's intended to be references related to that section or topic. Would something like `Issue references` or something that is more related to that section itself fit better?

Reply:
Right, I guess I typically don't need a local reference to a section when there are links to references integrated into the paragraphs. You may be able to get away with one Reference section.

Reply:
I'm inclined to leave it just for quickly navigating around, sort of like 'oh yeah, theres link here' type thing. Or when linking to a heading from another doc there is immediate contextual helpful links without too much going through the text if its not needed.

Reply:
👍